base/util.go

```diff
@@ -826,3 +826,21 @@ func ConvertBackQuotedStrings(data []byte) []byte {
 		return bytes
 	})
 }
+
+
+func ConvertToEmptyInterfaceSlice(i interface{}) (result []interface{}, err error) {
+
+	switch v := i.(type) {
+	case []string:
+		emptyInterfaceKeys := []interface{}{}
+		for _, key := range v {
+			emptyInterfaceKeys = append(emptyInterfaceKeys, key)
+		}
+		return emptyInterfaceKeys, nil
+	case []interface{}:
+		return v, nil
+	default:
```

Comment:
Are we sure that []string and []interface{} are the only cases we need to support?  No []int, non-slice, etc?